pkg/apis/kops/cluster.go

```diff
@@ -358,8 +358,27 @@ const (
 	LoadBalancerTypeInternal LoadBalancerType = "Internal"
 )
 
+// LoadBalancerClass string describes LoadBalancer classes (classic, network)
+type LoadBalancerClass string
+
+const (
+	LoadBalancerClassClassic     LoadBalancerClass = "Classic"
+	LoadBalancerClassNetwork     LoadBalancerClass = "Network"
+	LoadBalancerClassBothClassic LoadBalancerClass = "Both,Classic"
+	LoadBalancerClassBothNetwork LoadBalancerClass = "Both,Network"
```

Comment:
I was thinking the allowed values would be
```
Classic
Classic,Network
Network
Network,Classic
```
if multiple values are specified, the first value is what the DNS will point to